src/content/reference/react/useTransition.md

```diff
@@ -31,24 +31,24 @@ function TabContainer() {
 }
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lainnya dibawah ini.](#usage)
 
 #### Parameters {/*parameters*/}
 
-`useTransition` does not take any parameters.
+`useTransition` tidak menerima parameter apapun.
 
 #### Returns {/*returns*/}
 
-`useTransition` returns an array with exactly two items:
+`useTransition` mengembalikan array dengan tepat dua item:
```

Comment:
When we referenced by glossary. Array is `Senarai`. 

CMIIW @mhaidarhanif @resir014